integration_tests/pages/apply/appeal.ts

```diff
@@ -12,6 +14,15 @@ export default class AppealsPage extends Page {
     return new AppealsPage(application)
   }
 
+  static visitView(application: Application, appeal: Appeal): AppealsPage {
```

Comment:
This page class is representing two different pages. 
I think it would be best to create a new `appeals` folder in the `apply` folder. 
The pre-existing page should be a renamed `new` and the new one we're adding should be `show`